templates/endpoints_v2/README.md

```diff
@@ -0,0 +1,115 @@
+  api_key="NOT A REAL KEY",
+)
+
+# List all models.
+models = client.models.list()
+print(models)
+
+# Note: not all arguments are currently supported and will be ignored by the backend.
+chat_completion = client.chat.completions.create(
+    model="mistralai/Mistral-7B-Instruct-v0.1",
+    messages=[{"role": "system", "content": "You are a helpful assistant."}, {"role": "user", "content": "What are some of the highest rated restaurants in San Francisco?'."}],
+    temperature=0.01
+)
+
+print(chat_completion)
+
+```
+
+# Step 3 - Deploying a production service - TODO with new CLI
+
+To deploy an application with one model on an Anyscale Service you can run:
+
+```shell
+anyscale service rollout -f service.yaml --name {ENTER_NAME_FOR_SERVICE_HERE}
+```
+
+This is setup to run the Mistral-7B model, but can be easily modified to run any of the other models in this repo.
+
+# Step 4 - Query the service endpoint
+
+In order to query the endpoint, you can modify the `query.py` script, replacing the query url with the Service URL found in the Service UI.
+
+Note: please make sure to include the path "/v1" at the end of the Service url.
+
+Ansycale Services provide highly available fault tolerance for production LLM serving needs.  Learn more about [Anyscale Services](https://docs.anyscale.com/productionize/services/get-started)!
+
+# More Guides
+
+Endpoints makes it easy for LLM Developers to interact with OpenAI compatible APIs for their applications by providing an easy to manage backend for serving OSS LLMs.  It does this by:
+
+- Providing an extensive suite of pre-configured open source LLMs and embedding models, with defaults that work out of the box. 
+- Simplifying the addition of new LLMs.
+- Simplifying the deployment of multiple LLMs
+- Offering unique autoscaling support, including scale-to-zero.
+- Fully supporting multi-GPU & multi-node model deployments.
+- Offering high performance features like continuous batching, quantization and streaming.
+- Providing a REST API that is similar to OpenAI's to make it easy to migrate and integrate with other tools.
+
+Look at the following guides for more advanced use-cases -
+* [Deploy models for embedding generation](EmbeddingModels.md)
+* [Learn how to bring your own models](CustomModels.md)
+* [Deploy multiple LoRA fine-tuned models](DeployLora.md)
+* [Learn how to leverage different configurations that can optimize the latency and throughput of your models](OptimizeModels.md)
+* [Learn how to fully configure your deployment including auto-scaling, optimization parameters and tensor-parallelism](AdvancedModelConfigs.md)
+
+# Application Examples
+See examples of building applications with your deployed endpoint on the [Anyscale Endpoints](https://docs.endpoints.anyscale.com/category/examples) page.
+
+Be sure to update the api_base and token for your private deployment.  This can be found under the "Serve deployments" tab on the "Query" button when deploying on your Workspace.
```

Comment:
Not relevant for AIOA?

Reply:
Hmm.  I was referring to this sentence

> Be sure to update the api_base and token for your private deployment.  This can be found under the "Serve deployments" tab on the "Query" button when deploying on your Workspace.

Reply:
I can remove this. In AICA, we have a tab where we show how to query serve deployments on a workspace from outside